From: Luca Boccassi <bluca@debian.org>
Date: Mon, 6 Nov 2023 12:40:37 +0000
Subject: pkgconfig: keep unmerged paths for udevdir and systemdsystemunitdir

We are not ready to rebuild all packages installing system
units and udev rules to /usr/lib, so keep the old paths
for a while longer
---
 src/core/systemd.pc.in | 2 +-
 src/udev/udev.pc.in    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/core/systemd.pc.in b/src/core/systemd.pc.in
index 6cca2fa..51e2864 100644
--- a/src/core/systemd.pc.in
+++ b/src/core/systemd.pc.in
@@ -20,7 +20,7 @@ sysconfdir=${sysconf_dir}
 systemd_util_dir=${prefix}/lib/systemd
 systemdutildir=${systemd_util_dir}
 
-systemd_system_unit_dir=${prefix}/lib/systemd/system
+systemd_system_unit_dir=/lib/systemd/system
 systemdsystemunitdir=${systemd_system_unit_dir}
 
 systemd_system_preset_dir=${prefix}/lib/systemd/system-preset
diff --git a/src/udev/udev.pc.in b/src/udev/udev.pc.in
index cbf7693..d731642 100644
--- a/src/udev/udev.pc.in
+++ b/src/udev/udev.pc.in
@@ -11,5 +11,5 @@ Name: udev
 Description: udev
 Version: {{PROJECT_VERSION}}
 
-udev_dir={{UDEVLIBEXECDIR}}
+udev_dir=/lib/udev
 udevdir=${udev_dir}
